games-misc/xcowsay/xcowsay-1.4.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Displays a cute cow and message on your desktop"
+HOMEPAGE="
+	https://github.com/nickg/xcowsay
+	https://www.doof.me.uk/xcowsay/
+"
+SRC_URI="https://github.com/nickg/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dbus fortune nls"
+
+RDEPEND="
+	dev-libs/glib:2
+	x11-libs/gdk-pixbuf:2
+	x11-libs/gtk+:2
+	x11-libs/pango
+	dbus? (	dev-libs/dbus-glib )
+	fortune? ( games-misc/fortune-mod )
+"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="virtual/pkgconfig"
```

Comment:
Seems like you forgot a dep, 
```
 * Running eautoreconf in '/tmp/portage/games-misc/xcowsay-1.4/work/xcowsay-1.4' ...
 * Skipping 'autopoint --force' due autopoint not installed
 * Running aclocal -I m4 ...                                                     [ ok ]
 * Running autoconf --force ...                                                  [ !! ]

 * Failed Running autoconf !
 * 
 * Include in your bugreport the contents of:
 * 
 *   /tmp/portage/games-misc/xcowsay-1.4/temp/autoconf.out

 * ERROR: games-misc/xcowsay-1.4::testworld failed (prepare phase):
 *   Failed Running autoconf !
 * 
 * Call stack:
 *     ebuild.sh, line  125:  Called src_prepare
 *   environment, line 1516:  Called eautoreconf
 *   environment, line  915:  Called eautoconf '--force'
 *   environment, line  828:  Called autotools_run_tool '--at-m4flags' 'autoconf' '--force'
 *   environment, line  642:  Called die
 * The specific snippet of code:
 *           die "Failed Running $1 !";
 * 
 * If you need support, post the output of `emerge --info '=games-misc/xcowsay-1.4::testworld'`,
 * the complete build log and the output of `emerge -pqv '=games-misc/xcowsay-1.4::testworld'`.
 * The complete build log is located at '/tmp/portage/games-misc/xcowsay-1.4/temp/build.log'.
 * The ebuild environment file is located at '/tmp/portage/games-misc/xcowsay-1.4/temp/environment'.
 * Working directory: '/tmp/portage/games-misc/xcowsay-1.4/work/xcowsay-1.4'
 * S: '/tmp/portage/games-misc/xcowsay-1.4/work/xcowsay-1.4'

>>> Failed to emerge games-misc/xcowsay-1.4, Log file:

>>>  '/tmp/portage/games-misc/xcowsay-1.4/temp/build.log'
```
```
# cat /tmp/portage/games-misc/xcowsay-1.4/temp/autoconf.out
***** autoconf *****
***** PWD: /tmp/portage/games-misc/xcowsay-1.4/work/xcowsay-1.4
***** autoconf --force

configure.ac:16: error: possibly undefined macro: AM_GNU_GETTEXT
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:17: error: possibly undefined macro: AM_GNU_GETTEXT_VERSION
```

Could you check if there are some other missing deps? Looks good otherwise.

Reply:
Should be now ok. Added `gettext` to BDEPEND and `awk` to DEPEND.